Probesys/bileto
# dev: Move some docker files under docker/development

## Related issue(s)



N/A

## How to test manually



- start docker compose with `make docker-start`
- run the different commands under `docker/bin/`
- check everything works

## Checklist
